src/features/conversations/components/ConversationsContainer.tsx

```diff
@@ -0,0 +1,28 @@
+import { ConversationsList } from "./ConversationsList";
+import { useConversations } from "features/conversations/hooks/useConversations.hook";
+import useCurrentPatient from "fetchDataHooks/useCurrentPatient";
+import { Patient } from "models/patients/Patient.types";
+
+const ConversationsContainer = () => {
+  const { currentPatient: patient } = useCurrentPatient();
+
+  const { conversations, participants, isLoading, isError } = useConversations(
+    patient as Patient
+  );
+
+  if (isLoading) {
+    return <></>;
+  } else if (isError) {
+    return <></>;
+  } else {
+    return (
+      <ConversationsList
+        conversations={conversations ?? []}
+        participants={participants ?? {}}
+        pageSize={100}
+      />
+    );
+  }
+};
```

Comment:
Nitpick: but I find it a lot easier to read when the if/else statements are broken out into multiple if statements with an early return in them. Also you can simply return nul instead of <></>
```suggestion
  if (isLoading) {
    return null;
  }
  
  if (isError) {
    return null;
  }
  
  return (
    <ConversationsList
      conversations={conversations ?? []}
      participants={participants ?? {}}
      pageSize={100}
    />
  );
};
```

Reply:
Not nitpick: i think it's maybe slightly better UX to use a loading bar instead of an empty component when the state is `isLoading`

Reply:
Agreed! This code was copied from kariko where we had deferred the loading state until there were designs to use, @achidu do you know if there are loading state designs for capable care?

Reply:
Not yet, but can mock something up. Is there a standard loading component you can think of we can use easily?

Reply:
@LUMacKenn we currently use the MUI Skeleton component to simulate loading: https://mui.com/material-ui/react-skeleton/#main-content

Reply:
Ah, just saw this last comment @lvancraen. I had decided to use the circular progress from MUI as I know that's being used in a couple places in kariko as well. If it's ok with you, I'll get this out as is and I can revisit later, just want to have these changes in for Lawrence and myself to build off of